modin/core/execution/ray/common/engine_wrapper.py

```diff
@@ -108,6 +111,14 @@ def put(cls, data, **kwargs):
         ray.ObjectID
             Ray object identifier to get the value by.
         """
+        if isinstance(data, FunctionType):
```

Comment:
Will this work?
```suggestion
        if not isinstance(data, types.LambdaType):
```

Reply:
Unfortunately, it doesn't work. Look at the source code of the types module:
```python
LambdaType = type(lambda: None)         # Same as FunctionType
```